Makefile.core.mk

```diff
@@ -355,6 +355,9 @@ gen: go-gen mirror-licenses format update-crds gen-charts
 
 gen-check: gen check-clean-repo
 
+operator-proto:
```

Comment:
good go - although I think it makes more sense to remove the `Makefile.core.mk` and `Makefile` and all the other cruft from the repo merges.

Cheers
-steve